README.md

```diff
@@ -0,0 +1,55 @@
+# Configuration Service
+
+## Providers
+
+### localFile
+
+Configuration will be included in a local json file.
+
+_Example token: "./path/to/configuration"_
+
+### httpFile
+
+Configuration will be included in a json file on some external http/https host and will be served statically.
+
+_Example token: "[http://localhost:1234](http://localhost:1234)"_
+
+### hardcoreServer
+
+Configuration will be available as an entry point of a special "hardcore" server.
```

Comment:
I missed it in API review but we should not call it `hardcore` provider. We need to rename it. @ronenhamias do you have a name to suggest for this provider ?